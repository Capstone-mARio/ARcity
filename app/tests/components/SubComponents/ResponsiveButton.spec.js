import React from 'react';
import ResponsiveButton from '../../../components/SubComponents/ResponsiveButton';
import renderer from 'react-test-renderer';

test('renders ResponsiveButton correctly', () => {
  const tree = renderer.create(<ResponsiveButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
