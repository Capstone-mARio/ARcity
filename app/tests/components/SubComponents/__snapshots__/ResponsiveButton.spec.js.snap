// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ResponsiveButton correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FF69B4",
      "borderRadius": 5,
      "height": 86.42857142857142,
      "justifyContent": "center",
      "margin": 10,
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": 3,
        "width": 3,
      },
      "shadowOpacity": 0.4,
      "width": 700,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "rgba(255,255,255,1)",
        "fontFamily": "System",
        "fontSize": 20,
        "fontWeight": "600",
        "letterSpacing": 0.361328,
        "lineHeight": 28,
      }
    }
  />
</View>
`;
